CollapseLauncher/Classes/RegionManagement/FallbackCDNUtil.cs

```diff
@@ -81,7 +82,13 @@ private string GetRelativePathOnly(string url)
 
     internal static class FallbackCDNUtil
     {
-        private static HttpClient _client = new HttpClient()
+        private static readonly HttpClient _client = new HttpClient(new HttpClientHandler
+        {
+            AllowAutoRedirect = true,
+            UseCookies = true,
+            MaxConnectionsPerServer = 16,
```

Comment:
16 connections to CDN seems pretty overkill for me as the max it would download at a time is around 50MB for an update. To minimize any chance of the CDNs getting mad at us maybe setting it around 8 or even 6 should do just fine.